### DISCUSSION TITLE
Radicación automática de facturación electrónica de SINCO RE en SINCO SGD

### DISCUSSION THREAD

**Discussion by NestorFGG** [SINCOSOFT-SAS/SGD#34] [General]
Nuevo producto que optimizará el proceso de radicación y trámite de las facturas electrónicas recibidas por nuestro módulo SINCO RE a través de un proceso automático, así mismo, si dentro de los campos de la factura electrónica se recupera el código de barras de actas o entradas se podrán realizar los proceso de ejecución automática de las actividades del flujo programado en SGD relacionadas con asociación de documentos ADPRO y la generación de CP que llevará consigo la Aceptación de la factura electrónica con la emisión del evento a la DIAN y al proveedor.

**Comment by NestorFGG** [comment]
El desarrollo se encuentra en fase de pruebas y ajustes finales.

**Comment by NestorFGG** [comment]
El desarrollo pasa pruebas de esfuerzo satisfactoriamente.

**Comment by aleja550** [comment]
Este proyecto fue cancelado debido a ordenes corporativas, la alternativa a este desarrollo lo tiene el módulo de FE.
